solutions/src/test/java/ru/sberbank/school/task07/CollectionsTest.java

```diff
@@ -0,0 +1,79 @@
+package ru.sberbank.school.task07;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.Assert;
+
+import java.io.FileNotFoundException;
+import java.lang.reflect.Array;
+import java.util.*;
+
+public class CollectionsTest {
+
+    private String pathToFile;
+    private CollectionsServiceFactoryImpl csfi;
+    private FileParser fileParser;
+
+    @Before
+    public void initialize() {
+        pathToFile = "./src/test/java/ru/sberbank/school/task07/test.txt";
+        csfi = new CollectionsServiceFactoryImpl();
+        fileParser = new FileParserImpl();
+    }
+
+    @Test
+    public void testFileParser() throws FileNotFoundException {
+        initialize();
+        List<String> stringsFile = csfi.getFileParser().parse(pathToFile);
+        Assert.assertNotNull(stringsFile);
+        Assert.assertFalse(stringsFile.isEmpty());
+    }
+
+    @Test
+    public void testCounter() throws FileNotFoundException {
+        int count = csfi.getCounter(fileParser).count(pathToFile);
+        Assert.assertEquals(30, count);
+    }
+
+    @Test
+    public void testDifferentWordsImpl() throws FileNotFoundException {
+        pathToFile = "./src/test/java/ru/sberbank/school/task07/test2.txt";
+        Set set = (Set) csfi.getDifferentWords(fileParser).findSortedDifferentWords(pathToFile);
+        List actual = new ArrayList<>(set);
+        List expected = Arrays.asList("a", "b", "d", "cd", "dc", "qq", "svfdf");
+        Assert.assertEquals(expected, actual);
+    }
+
+    @Test
+    public void testWordsFrequency() throws FileNotFoundException {
+        Map actual = (Map) csfi.getWordFrequency(fileParser).countWords(pathToFile);
+        Map<String, Integer> expected = new HashMap<>();
+        expected.put("ду", 1); expected.put("ту", 3); expected.put("мся!", 1);
+        expected.put("ми", 1); expected.put("ко", 2); expected.put("зна", 2);
+        expected.put("по", 2); expected.put("й", 2); expected.put("ва", 2);
+        expected.put("да", 2); expected.put("зья", 1); expected.put("дру", 1);
+        expected.put("в", 1); expected.put("бавься", 1); expected.put("до", 1);
```

Comment:
Only one statement per line allowed.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a422549142>*